docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -451,6 +442,111 @@ should not have direct access to the underlying node.
 If you are deploying Teleport on a cluster of virtual machines, remove this line
 to run the SSH Service and enable secure access to the host.
 
+### Auth Service pool
+
+#### Open ports
+
+Ensure that, on each Auth Service instance, the following ports are open: 
+
+| Port | Description |
+| - | - |
+| <nobr>`3025`</nobr> | gRPC port to open to Proxy Service instances.|
+| `3026` | Optional port to open for traffic to the Teleport Kubernetes Service (if deploying on Kubernetes).|
```

Comment:
I don't understand this. By this logic, should we also add the desktop service port (3028) because you can deploy desktop service on auth instances as well?

(That's a rhetorical question - I believe the answer is no, I just don't understand why kubernetes_service is getting special treatment here)

Reply:
As above, this is based on the `teleport-cluster` chart, where Auth Service pods run the Kubernetes Service. My understanding is that Auth Service pods would run the Kubernetes Service to enable access to (at least) Teleport pods running on Kubernetes. Is that accurate, @hugoShaka ?

Reply:
Yeah, it's a bit complex and specific to the Helm chart. We packaged kube access at some point and have to live with this. We also cannot connect kube access to the proxy because we can't establish trust with the proxy without relying on external infra (LE, DNS, LoadBalancer) (discussed [here](https://github.com/gravitational/teleport/issues/19275#issuecomment-1477978729)). I tried to extract this service but failed, and users will be upset if we remove it.

tl;dr I would remove mention of the kube access port in a generic HA guide, this is an implementation detail because we made questionable choices

Reply:
👍 I've removed the Kubernetes mentions